### ISSUE TITLE
Font,Dock Reflections,iPhoneShadow don't work on iOS 9

### ISSUE THREAD

**Issue by maaya1989** [AnemoneTeam/Anemone#32]
I read very carefully on the wiki about all the functions but as I mentioned in title,font,dock reflections settings(blurred,no blurred etc),and iphoneshadow don't work on iOS9. I don't have an iOS 8 device now so I'm not sure if I did something wrong with font theming and dock settings(didn't install classic dock), but iphoneshadow has been confirmed not working while iphoneoverlay works properly.

FYI this is the code I added to info.plist

```
    <key>Font-SystemFont</key>
    <string>AvenirNext-UltraLight
</string>
    <key>Font-BoldSystemFont</key>
    <string>AvenirNext-Regular
</string>
```

Not working...

**Comment by coolstar** [comment]
Fonts aren't currently supported by Anemone. How broken are the dock reflections and shadows? A screenshot would be appreciated.

Also, please try making an icon semi-transparent or transparent so the shadow shows through in case the shadow does render but is hidden behind the icon and cut off by the mask on iOS 9.

**Comment by maaya1989** [comment]
dock settings have no effect...
![img_0003](https://cloud.githubusercontent.com/assets/13052129/10513093/7195ac70-7345-11e5-909c-54332488d11e.PNG)
![img_0004](https://cloud.githubusercontent.com/assets/13052129/10513094/726c3eac-7345-11e5-9445-ad2fe2c36f2f.PNG)

shadows I have tried something special too,no they simply don't show,it's not hidden or cut off...
![img_0005](https://cloud.githubusercontent.com/assets/13052129/10513239/995f14de-7346-11e5-8832-d81ffadc3b52.PNG)
![iphoneshadow](https://cloud.githubusercontent.com/assets/13052129/10513203/4b385270-7346-11e5-8dd7-15e291044703.png)

**Comment by coolstar** [comment]
Fixed in build 1.1-21.
